src/code42cli/cmds/cases.py

```diff
@@ -241,6 +244,10 @@ def add(state, case_number, event_id):
     """Associate a file event to a case, by event ID."""
     try:
         state.sdk.cases.file_events.add(case_number, event_id)
+    except Py42UpdateClosedCaseError:
+        raise
+    except Py42CaseAlreadyHasEventError:
+        raise
     except Py42BadRequestError:
```

Comment:
Since the above 2 custom exceptions are subclass of `Py42BadRequestError` had to catch and re-raise them to avoid displaying below error message.

Reply:
What cases causes the regular BadRequest to show up?  Do we really need to handle that?  I would expect a BadRequest to the more of the fault of the CLI rather than the User

Reply:
We should be able to remove the error handling enitirely here

Reply:
When the event-id or case-number are invalid it returns BadRequestError, if we add BadRequest in the groups list we would lose the custom messages we are showing to the user.

It will also affect `devices` and other feature where we are passing custom messages by catching BadRequest error

Reply:
I am definitely not suggestion add BadRequest to the groups list! I just wanted to make sure it was actually needed here